yb-voyager/src/migassessment/sizing.go

```diff
@@ -501,10 +560,9 @@ func checkFileExistsOnRemoteRepo(fileName string) bool {
 	}(resp.Body)
 
 	if resp.StatusCode != 200 {
-		//fmt.Println("File does not exist on remote location")
 		return false
 	} else {
-		downloadPath := strings.ReplaceAll(fileName, "resources/", baseDownloadPath)
+		downloadPath := assessmentMetadataDir + fileName
 		out, _ := os.Create(downloadPath)
 		defer func(out *os.File) {
 			err := out.Close()
```

Comment:
don't completely ignore the error returned by `io.Copy()`
Even in case you are handling that error, its good to add as a Warn or Infof, in logs, so that we know why that decision was made.